src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -50,7 +48,6 @@ public PersonCard(Person person, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
```

Comment:
Is there a reason why we are removing address from the person card?

Reply:
@dickongwd We discussed earlier that since we will be adding a more detailed profile section, the `PersonCard` should only contain the more essential fields.

Reply:
On that note, I wonder if we should even keep the other fields like `phone and email`.